tests/test_PsfFlux.py

```diff
@@ -32,6 +32,22 @@
                                   SingleFramePluginTransformSetupHelper)
 
 
+def compute_chi2(exposure, centroid, instFlux, maskPlane=None):
+    """Return the chi2 for the exposure PSF at the given position."""
```

Comment:
More documentation? E.g., it's not obvious right away what `maskPlane` is, and you don't mention returning the pixel count.